bundles/config.en_us.json

```diff
@@ -6,9 +6,10 @@
     "ripe_commons.modal.apply": "Apply",
     "ripe_commons.modal.select": "Select",
 
-    "ripe_commons.pickers.undo": "Undo",
-    "ripe_commons.pickers.limited": "This option is limited to specific material or colour combinations.",
-    "ripe_commons.pickers.back": "Do you want to go back?",
+    "ripe_commons.restrictionsAlert.restrictionsAlert": "Undo",

```

Comment:
Should be `restrictions_alert` (look a few lines below and you'll find `add_initials`)